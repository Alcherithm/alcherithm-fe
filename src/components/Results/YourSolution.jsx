import React from 'react';
import PropTypes from 'prop-types';

const YourResults = () => (

);

YourResults.propTypes = {

};

export default YourResults