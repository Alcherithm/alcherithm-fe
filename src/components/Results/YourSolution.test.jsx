import React from 'react';
import { shallow } from 'enzyme';
import YourSolution from './YourSolution.jsx';

describe('the YourSolution Component', () => {
  it('renders YourSolution', () => {
    const wrapper = shallow(<YourSolution userSolution="user solution here"/>);
    expect(wrapper).toMatchSnapshot();
  });
});

