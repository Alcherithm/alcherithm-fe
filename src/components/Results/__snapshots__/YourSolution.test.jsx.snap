// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the YourSolution Component renders YourSolution 1`] = `
<section
  className="YourSolution"
>
  user solution here
</section>
`;
